Readme.md

```diff
@@ -83,9 +83,9 @@ Authors:
 * [TAImatem](https://github.com/TAImatem)
 
 Special thanks to:
-* [Outer Wilds](http://www.outerwilds.com)
-* [Outer Wilds on Discord](https://discord.gg/csKYR3w)
-* [Outer Wilds on Reddit](https://www.reddit.com/r/outerwilds)
+* [Outer Wilds](http://www.outerwilds.com
```

Comment:
missing bracket

Reply:
What how. Ugh I'll fix it tomorrow, or close this and you can edit it.